I saw that, but isn't that an infrastructure problem, some issue with Jenkins etc.?

For me, the tests are all failing locally even on the main/ branch, with an error that

> Cannot find module '@carbon/feature-flags' from 'packages/react/src/components/FeatureFlags/index.tsx'

That may be unrelated though.

--- a/packages/react/src/components/Tile/Tile-test.js
+++ b/packages/react/src/components/Tile/Tile-test.js
@@ -111,12 +111,7 @@ describe('Tile', () => {
       const onClick = jest.fn();
       render(
         <div role="group" aria-label="selectable tiles">
-          <SelectableTile
-            disabled
-            id="tile-1"
-            name="tiles"
-            onClick={onClick}
-            value="value">
+          <SelectableTile disabled id="tile-1" onClick={onClick}>
             <span role="img" aria-label="vertical traffic light">
               🚦
             </span>
@@ -130,25 +125,13 @@ describe('Tile', () => {
     it('should cycle elements in document tab order', async () => {
       render(
         <div role="group" aria-label="selectable tiles">
-          <SelectableTile
-            data-testid="element"
-            id="tile-1"
-            name="tiles"
-            value="value">
+          <SelectableTile data-testid="element" id="tile-1">
             tile 1
           </SelectableTile>
-          <SelectableTile
-            data-testid="element"
-            id="tile-2"
-            name="tiles"
-            value="value">
+          <SelectableTile data-testid="element" id="tile-2">
             tile 2
           </SelectableTile>
-          <SelectableTile
-            data-testid="element"
-            id="tile-3"
-            name="tiles"
-            value="value">
+          <SelectableTile data-testid="element" id="tile-3">
             tile 3
           </SelectableTile>
         </div>
@@ -180,11 +163,7 @@ describe('Tile', () => {
 
     it('should respect slug prop', () => {
       render(
-        <SelectableTile
-          slug={<AILabel />}
-          id="tile-1"
-          name="tiles"
-          value="value">
+        <SelectableTile slug={<AILabel />} id="tile-1">
           Default tile
         </SelectableTile>
       );
@@ -440,7 +419,7 @@ describe('Tile', () => {
 
   it('respect selected prop', async () => {
     const { container } = render(
-      <SelectableTile id="selectable-tile-1" selected value={'test'}>
+      <SelectableTile id="selectable-tile-1" selected>
         Option 1
       </SelectableTile>
     );

--- a/packages/react/src/components/Tile/Tile.tsx
+++ b/packages/react/src/components/Tile/Tile.tsx
@@ -561,7 +561,10 @@ SelectableTile.propTypes = {
    * The `name` of the `<input>`.
    * @deprecated
    */
-  name: PropTypes.string,
+  name: deprecate(
+    PropTypes.string,
+    'The `name` property is no longer used.  It will be removed in the next major release.'
+  ),
 
   /**
    * The empty handler of the `<input>`.
@@ -602,7 +605,10 @@ SelectableTile.propTypes = {
    * The value of the `<input>`.
    * @deprecated
    */
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+  value: deprecate(
+    PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    'The `value` property is no longer used.  It will be removed in the next major release.`'
+  ),
 };
 
 export interface ExpandableTileProps extends HTMLAttributes<HTMLDivElement> {

